mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/reports/viewpagerfragments/SubmittedReportsFragment.kt

```diff
@@ -67,7 +67,7 @@ class SubmittedReportsFragment : BaseBindingFragment<FragmentReportsListBinding>
                 ReportsUtils.showReportDeletedSnackBar(
                     getString(
                         R.string.Report_Deleted_Confirmation, it
-                    ),baseActivity
+                    ), baseActivity
                 )
                 viewModel.listOutbox()
```

Comment:
After the report is deleted, this allways list outbox reports (even when the submitted report is deleted).